### PR TITLE
Get rid of bash (scenario 1.5 and 3.5)

### LINKED ISSUE #200: Bashless scenario 1.5
Part of #157

### LINKED ISSUE #207: Bashless scenario 3.5

### LINKED ISSUE #211: Use different workspaces for testing and running from awx
Let's use `awx` workspace on AWX

### PR DESCRIPTION
**Pull request by Polina-Gubina**
bashless scenario1_5 and scenario3_5

Resolve #200
Resolve #207
Closes #211

### CODE REVIEW

File: playbooks/scenario1_5_setup.yml [hunk 1]
```diff
@@ -1,11 +1,46 @@
...
+    scenario_name: "scenario1_5"
+    key_name: "key_scenario1_5"
+    requirements: "requirements.txt"
+    infra_state: present
+
+- name: Setting terraform outputs
+  hosts: local
+  become: yes
+  vars:
+    bastion_public_ip: "{{ tf_output.outputs[\"out-scn1_5_lb_fip\"].value }}"
```
Comment (lego963) [inline playbooks/scenario1_5_setup.yml]: Bastion public IP = out-scn1_5_lb_fip
Loadbalancer public IP = out-scn1_5_bastion_fip
Is it ok?

Comment (Polina-Gubina) [reply playbooks/scenario1_5_setup.yml]: tf_output.outputs[\"out-scn1_5_lb_fip\"].value gives  value out-scn1_5_lb_fip of dictionary  tf_output.outputs

Comment (Polina-Gubina) [reply playbooks/scenario1_5_setup.yml]: i am sorry,
 you are right

File: playbooks/scenario1_5_setup.yml [hunk 2]
```diff
@@ -1,11 +1,16 @@
 ---
-- name: Register existing settings
+- name : Register existing settings and hosts
   hosts: local
+  become: yes
   tasks:
+    - name: Setting variables terraform outputs
+      set_fact:
+        BASTION_PUBLIC_IP: "{{ bastion_public_ip }}"
```
Comment (outcatcher) [inline playbooks/scenario1_5_setup.yml]: why make variable upper case?

File: scenarios/scenario3_5/infrastructure.tf [hunk 3]
```diff
@@ -3,7 +3,7 @@ locals {
   workspace_prefix = terraform.workspace == "default" ? "" : "${terraform.workspace}-"
   key_pair = {
     public_key = var.public_key
-    key_name   = "${local.workspace_prefix}kp_${var.scenario}"
+    key_name   = "${local.workspace_prefix}kl_${var.scenario}"
```
Comment (lego963) [inline scenarios/scenario3_5/infrastructure.tf]: is it required?

File: scenarios/scenario3_5/resources/vm.tf [hunk 4]
```diff
@@ -104,3 +104,11 @@ resource "opentelekomcloud_networking_port_v2" "initiator_instance_port" {
     ip_address = "${var.net_address}.11"
   }
 }
+
+output "target_instance_id" {
```
Comment (outcatcher) [inline scenarios/scenario3_5/resources/vm.tf]: `_ip`, not `_id`

File: scenarios/scenario2/variables.tf [hunk 5]
```diff
@@ -12,6 +7,6 @@ variable "ecs_image" {}
 variable "scenario" {}
 variable "psql_version" {}
 variable "psql_port" {}
-variable "psql_password" {}
+variable "psql_password" { default = "qwerty"}
```
Comment (outcatcher) [inline scenarios/scenario2/variables.tf]: `qwerty` won't be valid, there are password sec restrictions, so creation of RDS will fail

File: playbooks/scenario1_5_setup.yml [hunk 6]
```diff
@@ -1,62 +1,118 @@
...
+      ansible_ssh_user: "linux"
+      ansible_ssh_private_key_file: "{{ local_private_key }}"
+  - name: Register nodes
+    add_host:
+      name: "{{ item }}"
+      groups: "gatewayed"
+      ansible_ssh_user: "linux"
+      ansible_ssh_private_key_file: "{{ local_private_key }}"
+      ansible_ssh_common_args: "-o ProxyCommand='ssh -W %h:%p -q linux@{{ bastion_public_ip }} -i {{ local_private_key }}' -o PasswordAuthentication=no"
+    loop: "{{ node_ips }}"
```
Comment (anton-sidelnikov) [inline playbooks/scenario1_5_setup.yml]: you can use here loop: "{{ hostvars.test_host.node_ips }} and not define variable node_ips, as in scenario1_setup.yml

File: scenarios/scenario2/variables.tf (hunk 5 above)
Comment (anton-sidelnikov) [reply scenarios/scenario2/variables.tf]: qwerty12345! will fit

File: scenarios/scenario3_5/infrastructure.tf (hunk 3 above)
Comment (anton-sidelnikov) [reply scenarios/scenario3_5/infrastructure.tf]: should be "${local.workspace_prefix}kp_${var.scenario}

File: inventory/prod/group_vars/credentials.yml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+credentials_env:
+  AWS_ACCESS_KEY_ID: "{{ lookup('env', 'AWS_ACCESS_KEY_ID') }}"
+  AWS_SECRET_ACCESS_KEY: "{{ lookup('env', 'AWS_SECRET_ACCESS_KEY') }}"
+  INFLUX_PASSWORD: "{{ influxdb_user_password }}"
```
Comment (lego963) [inline inventory/prod/group_vars/credentials.yml]: is this variable could be find without lookup `influxdb_user_password`?

Comment (outcatcher) [reply inventory/prod/group_vars/credentials.yml]: It is provided by AWX

File: playbooks/scenario3_5_setup.yml [hunk 8]
```diff
@@ -1,15 +1,93 @@
 ---
+- name: Prepare clouds.yaml for test_host
+  import_playbook: prepare_config.yml
+
+- name: Prepare variables
+  hosts: test_host
+  tasks:
+  - name: Register scenario 2 vars
```
Comment (lego963) [inline playbooks/scenario3_5_setup.yml]: scenario2 vars?

Comment (Polina-Gubina) [reply playbooks/scenario3_5_setup.yml]: that is a misprint, sorry

**Pull request merged**